tests/containers/buildah.pm

```diff
@@ -26,8 +26,8 @@ use Utils::Backends qw(is_svirt);
 
 sub run_tests {
     my $runtime = shift;
-
     my $image = get_var("CONTAINER_IMAGE_TO_TEST", "registry.opensuse.org/opensuse/tumbleweed:latest");
+    record_info('buildah info', script_output("buildah info", timeout => 300));
```

Comment:
The same, please remove the `timeout` parameter

Reply:
In the worst case scenario, the test will be hanging on an output that should be instant for 5 minutes. That is simply not acceptable given the amount of workload that we have

Reply:
As I told https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/22357#discussion_r2145021972, during my work and many VR runs, I got the issue of tests remaining blocked for more than 1hour on the same point, therefore I cosnidered timeouts a key feature to add here.

I was even considering useful to add timeouts in the `validate_script_...` commands.

Anyway, ...

Reply:
The problem is that the default timeout is 90, not 3150:

https://github.com/os-autoinst/os-autoinst/blob/master/testapi.pm#L836

You got this 3150 value from the multiplication of 90 with `TIMEOUT_SCALE=35` used by the job above.  You were thus increasing - not decreasing - the effective timeout, making any eventuality worse by having the job hanging for 3150 * 35 = 110250 seconds.  A full 30 hours.

Most timeouts specified in the testapi functions are multiplied by `TIMEOUT_SCALE`:

https://github.com/os-autoinst/os-autoinst/blob/master/testapi.pm#L91-L94